indico/web/client/styles/modules/_papers.scss

```diff
@@ -243,10 +243,14 @@
     .paper-metadata-dt {
         border-bottom: 1px dotted $gray;
         color: $gray;
-        margin: 0 3px 0 3px;
+        margin: 0 3px;
 
         &:hover {
             color: $light-black;
         }
     }
 }
+
+.paper-timeline {
+    min-height: 500px;
```

Comment:
Why do we need this?

Reply:
To make the `Loader` look nice when fetching the paper-related data ;)

Reply:
By nicer I mean centered

Reply:
I see now, cool :)